addons/mail/static/src/messaging/component/chatter_topbar/chatter_topbar.xml

```diff
@@ -4,22 +4,25 @@
     <t t-name="mail.messaging.component.ChatterTopbar" owl="1">
         <div class="o_ChatterTopbar">
             <t t-if="chatter">
-                <div class="o_ChatterTopbar_actions">
+                <div class="o_ChatterTopbar_actions"
+                     t-att-class="{'o-has-active-button': chatter.isComposerVisible }">
```

Comment:
keep on the same line here

(note: if multi line, the closing `>` should be on new line matching indent of open div, and new line attributes with 4 spaces (not aligned on top attribute)))